torch/csrc/jit/script/compiler.cpp

```diff
@@ -332,9 +355,13 @@ struct to_ir {
             environment_stack->setVar(name, graph->addInput(name));
           }
           break;
-        case TK_EXPR_STMT:
-          emitExpr(ExprStmt(stmt).expr(), 0);
-          break;
+        case TK_EXPR_STMT: {
+          auto exprs = ExprStmt(stmt).exprs();
+          for (const auto& expr : exprs) {
+            emitExpr(expr, {0, false});
```

Comment:
It would be nice to allow constructing `CallsiteDescriptor`s from an `int`, since it's usually not the case that we allow variable number of outputs